app/pages/reservationCreate/ReservationCreatePageContainer.js

```diff
@@ -26,7 +26,9 @@ export function UnconnectedReservationCreatePageContainer(props) {
       <h1>Varauksen tekeminen</h1>
       <Loader loaded={Boolean(props.resource)}>
         <ReservationCreateForm
+          begin={props.location.query.begin}
```

Comment:
-  [ ] I think now that this `begin` prop has been added, `date` prop is not needed anymore. So you could probably remove this line and replace `date` with `begin` on the next line (so that `begin` always has some value). Then you could remove `date` from `ReservationCreateFormContainer`.